app/Console/Commands/CheckinLicensesFromAllUsers.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\LicenseSeat;
+use Illuminate\Console\Command;
+use App\Models\User;
+use App\Models\License;
+use Illuminate\Database\Eloquent\Model;
+
+class CheckinLicensesFromAllUsers extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'snipeit:checkin-from-all {--license_id=} {--notify}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Checks in licenses from all users';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+
+        $license_id = $this->option('license_id');
+        $notify = $this->option('notify');
+
+        if (!$license_id) {
+            $this->error('ERROR: License ID is required.');
+            return false;
+        }
+
+
+        if (!$license = License::where('id','=',$license_id)->first()) {
+            $this->error('Invalid license ID');
+            return false;
+        }
+
+        $this->info('Checking in ALL seats for '.$license->name);
+
+
+        $licenseSeats = LicenseSeat::where('license_id', '=', $license_id)
+            ->whereNotNull('assigned_to')
+            ->with('user')
+            ->get();
+
+        $this->info(' There are ' .$licenseSeats->count(). ' seats checked out: ');
+
+        if (!$notify) {
+            $this->info('No mail will be sent.');
+        }
+
+        foreach ($licenseSeats as $seat) {
+            $this->info($seat->user->username .' has a license seat for '.$license->name);
+
+            if (!$notify) {
+                $seat->user->email = null;
```

Comment:
Are we sure we won't somehow accidentally end up saving the user's email as `null` here?

Reply:
It doesn't seem to be that way?

<img width="837" alt="Screen Shot 2020-08-14 at 3 21 35 PM" src="https://user-images.githubusercontent.com/197404/90297019-1634fa80-de42-11ea-922f-15ddb60e33b2.png">

<img width="1291" alt="Screen Shot 2020-08-14 at 3 21 44 PM" src="https://user-images.githubusercontent.com/197404/90297036-1df49f00-de42-11ea-8519-9a1b5ca8a17b.png">

That said, I've moved it until after the `save()` just to be sure. As long as it comes before the notification, it should accomplish the same goal.